### ISSUE TITLE
Log learning rate changes

### ISSUE THREAD

**Issue by nitrif** [Microsoft/CNTK#2510]
Is it possible to log learning rate / momentum changes to the log file? I find it very incovinient not beeing able to know what parameters were in use at the specific training stage.

Thanks,
Nikola

**Comment by frankibem** [comment]
I haven't tested this with momentum, but ProgressPrinter prints the new learning rate when it changes when I use a learning_parameter_schedule with a Trainer.

**Comment by nitrif** [comment]
You are right. I have used depricated learning_parameter_schedule. But, I just checked that momentum is not logged with learning_parameter_schedule

I would pref:

- to log momentum

- to log parameters at the start of each epoch, not only on change

because sometimes after checkpoint restore i have no idea what is going on.

Thanks,
Nikola
